core/app/crash/reporting/encoder.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reporting
+
+import (
+	"bytes"
+	"io"
+	"mime/multipart"
+)
+
+type encoder struct {
+	appName    string
+	appVersion string
+	osName     string
+	osVersion  string
+	stacktrace string
+}
+
+const multipartBoundary = "::multipart-boundary::"
```

Comment:
Is there a reason we set this manually? Typically a UUID is desired I think.

Reply:
I made it fixed for testing. I figured the chances of `"::multipart-boundary::"` being used in the stack trace or other data was slim to none, so altering the boundary string on tests was fairly pointless.